data/apparmor/usr.sbin.httpd-prefork

```diff
@@ -1,10 +1,11 @@
 # Last Modified: Thu Mar 21 15:39:34 2019
 #include <tunables/global>
 
-/usr/sbin/httpd-prefork {
+/usr/sbin/httpd-prefork flags=(attach_disconnected) {
```

Comment:
https://openqa.opensuse.org/tests/1843415#step/apache2_changehat/96

interesting - this somehow ended up on the command line, where it does not work for not being escaped properly

Reply:
openQA has its own code to parse AppArmor profiles? Great :-/

apparmor_parser is more familiar with this job ;-) therefore I just pushed another commit that drops the custom code in `get_named_profile()` and replaces it with a call to `apparmor_parser`. (Please read the commit message, you'll enjoy it!)

Of course this change is untested ;-)